src/main/resources/cloudformation/base.yaml

```diff
@@ -50,8 +50,9 @@ Resources:
                 - !GetAtt 'CmsIamRole.Arn'
             Resource:
               - !Join ['', ['arn:aws:s3:::', !Ref 'CerberusConfigBucket', /certificates/*]]
-              - !Join ['', ['arn:aws:s3:::', !Ref 'CerberusConfigBucket', cms/*]]
-              - !Join ['', ['arn:aws:s3:::', !Ref 'CerberusConfigBucket', cms]]
+              - !Join ['', ['arn:aws:s3:::', !Ref 'CerberusConfigBucket', /certificates]]
```

Comment:
If you wait till I get back to merge this PR that is fine, how ever this is needed in highlander now or else new environments will be broken.